providers/aws/components/apigateway/setup.ftl

```diff
@@ -458,15 +458,21 @@
                     valueIfTrue(
                         {
                             "RegionalCertificateArn":
-                                getArn(value["domain"].CertificateId, true, regionId),
+                                ( getArn(value["domain"].CertificateId, true, regionId) )?has_content?then(
```

Comment:
contentIfContent(
  getArn(value["domain"].CertificateId, true, regionId) ), 
  "COTFatal: Could not find certificate " + value["domain"].CertificateId
)

means function only called once mostly

Reply:
Fixed